content/pages/photography-policy.md

```diff
@@ -0,0 +1,24 @@
+Title: Photography Policy
+Category: pages
+Status: hidden
+Slug: photography-policy
+
+We often take pictures at events, but not without consent.
+We will not take you picture if you're wearing a lanyard (We keep
+some lanyards available at all our events) or if you request that we
+don't take your picture.
+
+We can use these pictures for our social media profiles (Facebook,
+Twitter etc.) and also for our publicity material like brochures.
+
+We always tell everyone at the beginning of the event that photos
```

Comment:
"We will always tell" -> "We will review the photography policy at the beginning of every event we run"